repos/system_upgrade/el7toel8/actors/networkmanagerreadconfig/libraries/library.py

```diff
@@ -0,0 +1,55 @@
+import os
+
+from six.moves.configparser import ConfigParser, ParsingError
+
+from leapp.libraries.stdlib import CalledProcessError, run, api
+
+
+def _is_file_readable(path):
+    ''' Verify if file exists and is readable '''
+    return os.path.isfile(path) and os.access(path, os.R_OK)
+
+
+def read_nm_config(file_path=None):
+    # import pdb
```

Comment:
Please remove the commented out code.

Reply:
btw try ipdb, it looks much better and has autocompletion :)

Reply:
yeah @fernflower ! nice catch! tks :)